configure

```diff
@@ -76,56 +76,30 @@ then
   fi
 fi
 
-NORMALIZED=""
-mynormalizepath () {
-  # Normalize PREFIX e.g.
-  P="$1"
-  case "$P" in
-    *~*)
-      # Replace ~ with home directory
-      P="${P/#\~/$HOME}"
-      ;;
-    *)
-      ;;
-  esac
-
-  if [ ! -d $P ]
-  then
-    echo "Prefix directory does not exist, creating directory."
-    mkdir -p $P
-  fi
-
-  # Normalize $P more - normalize path
-  P=`(unset CDPATH && cd "$P" && pwd)`
-
-  # Make sure we can cd into $P
-  if [ $? -ne 0 ]
-  then
-    echo "Error: Could not cd into prefix directory '$P'"
-    echo "       ... perhaps it does not exist?"
-    exit -1
-  fi
-
-  NORMALIZED="$P"
-  return 0
-}
-
 # Save configured installation prefix/dir
 rm -f configured-chpl-home
 rm -f configured-prefix
 CONFIGURED=""
 if [ "$DEST_DIR_SET" -ne 0 ]
 then
-  # Normalize DEST_DIR
-  mynormalizepath "$DEST_DIR"
-  DEST_DIR="$NORMALIZED"
+  case $DEST_DIR in 
+    (/*) pathchk -- "$1";; 
+    (*) 
+    echo "error: expected an absolute directory name for --chpl-home: $DEST_DIR"
```

Comment:
PR message says

> We still need to give errors in cases where the path would be problematic.

But I think this covers it. Can you update the PR message to say the PR does this? (The current wording sounds like future work to me).

Reply:
Ah, thanks for catching that. I created the PR message before the second commit and forgot to update it 😅